From: Stephen Gelman <ssgelm@debian.org>
Date: Tue, 29 Jan 2019 06:48:00 +0000
Subject: Disable tests requiring internet access

Forwarded: not-needed
---
--- a/endpoint_test.go
+++ b/endpoint_test.go
@@ -87,6 +87,7 @@
 }
 
 func TestNewEndpointFailsDueToLookupIP(t *testing.T) {
+	t.Skip("skip test requiring internet access")
 	_, err := zipkin.NewEndpoint(serviceName, unreachableHostPort)
 
 	if err == nil {
--- a/middleware/http/client_test.go
+++ b/middleware/http/client_test.go
@@ -10,6 +10,7 @@
 )
 
 func TestHTTPClient(t *testing.T) {
+	t.Skip("skip test requiring internet access")
 	reporter := recorder.NewReporter()
 	defer reporter.Close()
 
--- a/span_test.go
+++ b/span_test.go
@@ -31,6 +31,7 @@
 }
 
 func TestRemoteEndpoint(t *testing.T) {
+	t.Skip("skip test requiring internet access")
 	tracer, err := NewTracer(reporter.NewNoopReporter())
 	if err != nil {
 		t.Fatalf("expected valid tracer, got error: %+v", err)
